<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>991</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <underline>true</underline>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>CRUD</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>391</width>
        <height>331</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(239, 246, 255);
</string>
      </property>
      <property name="title">
       <string>Reservation hebergement</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_nomPrenom">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>80</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{

border: 2px solid rgb(76,84,140);

border-radius: 10px;

color: #FFF;

padding-left:20px;

padding-right:20px;

background-color:rgb(255 255, 255)

}

QLineEdit:hover{

border:2px sold ;

color: rgb(255, 0, 0);

}

QLineEdit:focus{

border:20px sold rgb(0, 0, 0);

background-color:rgb(131,131,131);

}



</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_hebergement">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>120</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{

border: 2px solid rgb(76,84,140);

border-radius: 10px;

color: #FFF;

padding-left:20px;

padding-right:20px;

background-color:rgb(255, 255, 255)

}

QLineEdit:hover{

border:2px sold ;

color: rgb(255, 0, 0);

}

QLineEdit:focus{

border:20px sold rgb(0, 0, 0);

background-color:rgb(131, 131, 131);

}



</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_prix">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>240</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{

border: 2px solid rgb(76,84,140);

border-radius: 10px;

color: #FFF;

padding-left:20px;

padding-right:20px;

background-color:rgb(255, 255, 255)

}

QLineEdit:hover{

border:2px sold ;

color: rgb(255, 0, 0);

}

QLineEdit:focus{

border:20px sold rgb(0, 0, 0);

background-color:rgb(131, 131, 131);

}



</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_checkin">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>160</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);

border: 2px solid rgb(76,84,140);

border-radius: 10px;

color: #FFF;

padding-left:20px;

padding-right:20px;

background-color:rgb(0, 0, 127)

}

QLineEdit:hover{

border:2px sold ;

color: rgb(255, 0, 0);

}

QLineEdit:focus{

border:20px sold rgb(0, 0, 0);

background-color:rgb(131, 131, 131);

}



</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_checkout">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>200</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="nomPrenom">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
         <underline>true</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text">
        <string>Nom et Prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="hybergement">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>130</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>true</italic>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Hybergement</string>
       </property>
      </widget>
      <widget class="QLabel" name="checkin">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>true</italic>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Check_in</string>
       </property>
      </widget>
      <widget class="QLabel" name="checkout">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>200</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>true</italic>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Check_out</string>
       </property>
      </widget>
      <widget class="QLabel" name="prix">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>240</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>true</italic>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
      <widget class="QLabel" name="Id">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>true</italic>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Id</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Id">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>40</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{

border: 2px solid rgb(76,84,140);

border-radius: 10px;

color: #FFF;

padding-left:20px;

padding-right:20px;

background-color:rgb(255 255, 255)

}

QLineEdit:hover{

border:2px sold ;

color: rgb(255, 0, 0);

}

QLineEdit:focus{

border:20px sold rgb(0, 0, 0);

background-color:rgb(131,131,131);

}



</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_ajoutcl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>ajouter un client</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_modifcl">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>290</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>modifier un client</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supcl">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>290</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>supprimer un client</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_rh">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>0</y>
        <width>651</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>metiers</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>401</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(239, 246, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_statistiques">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Statistiques</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_exportpdf">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Export pdf</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_triR">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>0</y>
        <width>611</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rechercheId">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>rechercheId</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rechercheN">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>rechercheN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_triId">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>TriId</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_triN">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>TriN</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_rechercheId">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{

border: 2px solid rgb(76,84,140);

border-radius: 10px;

color: #FFF;

padding-left:20px;

padding-right:20px;

background-color:rgb(255 255, 255)

}

QLineEdit:hover{

border:2px sold ;

color: rgb(255, 0, 0);

}

QLineEdit:focus{

border:20px sold rgb(0, 0, 0);

background-color:rgb(131,131,131);

}
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_rechercheN">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{

border: 2px solid rgb(76,84,140);

border-radius: 10px;

color: #FFF;

padding-left:20px;

padding-right:20px;

background-color:rgb(255 255, 255)

}

QLineEdit:hover{

border:2px sold ;

color: rgb(255, 0, 0);

}

QLineEdit:focus{

border:20px sold rgb(0, 0, 0);

background-color:rgb(131,131,131);

}
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_triP">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>TriPrix</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_alerte">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>280</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Offre</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>statisques</string>
     </attribute>
     <widget class="QCustomplot" name="customplot" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>571</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Calendrier</string>
     </attribute>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1</y>
        <width>421</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_choisir">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>180</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Choisir date</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_reserve">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>120</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomplot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
